### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andrewy-splunk**
Folded README.MD from old repo into this one - let me know if there are any issues.

### CODE REVIEW

**Review by andrewy-splunk — COMMENTED** [review]
updated headers

File: README.md [hunk 1]
```diff
@@ -103,17 +101,50 @@ require additional configurations applied to
 ### Prerequisites
 
 The following prerequisites are required to use the helm chart:
+- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported) 
+- See https://github.com/kubernetes/helm for more information.
+- Splunk Enterprise 7.0 or later.
+- A minimum of one Splunk platform index ready to collect the log data. This index will be used for ingesting logs.
+- An HTTP Event Collector (HEC) token and endpoint. See the following topics for more information:
+  * https://docs.splunk.com/Documentation/Splunk/8.2.0/Data/UsetheHTTPEventCollector
+  * https://docs.splunk.com/Documentation/Splunk/8.2.0/Data/ScaleHTTPEventCollector
```
Comment (rockb1017) [inline README.md:110]: this is required only when they are sending data to Splunk platform (SE or cloud). they can only send data to Olly suite.

Comment (rockb1017) [reply README.md:110]: let's add back 
To send data to [Splunk Enterprise/Cloud](https://docs.splunk.com/Documentation/Splunk/8.2.2/Data/UsetheHTTPEventCollector)

File: README.md [hunk 2]
```diff
@@ -103,17 +101,50 @@ require additional configurations applied to
...
+- To send data to [Splunk Observability Cloud:](https://docs.splunk.com/Observability/gdi/opentelemetry/install-k8s.html)
```
Comment (rockb1017) [inline README.md:111]: move ":" out.

File: README.md [hunk 3]
```diff
@@ -103,17 +101,50 @@ require additional configurations applied to
...
+```
+
+To learn more about using and modifying charts, see:
+* https://github.com/splunk/sck-otel/tree/main/charts
+* https://docs.helm.sh/using_helm/#using-helm.
+
+
+### Configuration variables for Helm
+
+The default values file can be found here [default values file](https://github.com/splunk/sck-otel/blob/main/charts/sck-otel/values.yaml)
```
Comment (rockb1017) [inline README.md:146]: this part is referencing to old repository. it needs to be updated to this repo.

actually, we already have a simple version of helm install command example for o11y suite. Line # 169 of your branch. 
let's add one in similar format. 

```
helm install my-splunk-otel-collector --set="splunkPlatform.endpoint=127.0.0.1:8088,splunkPlatform.token=xxxxxx,splunkPlatform.metrics_index=k8s-metrics,splunkPlatform.index=main,clusterName=my-cluster" splunk-otel-collector-chart/splunk-otel-collector
```

Comment (rockb1017) [reply README.md:146]: and delete this whole part.

File: README.md [hunk 4]
```diff
@@ -162,11 +193,181 @@ $ helm delete my-splunk-otel-collector
...
+|                   1 |              27,000 |               256 |             1.6 |    27,000 |
+|                   1 |              49,000 |               256 |             1.8 |    30,000 |
+|                   1 |              49,000 |               516 |             1.8 |    28,000 |
+|                   1 |              49,000 |              1024 |             1.8 |    24,000 |
+|                   2 |              20,000 |               256 |             1.3 |    20,000 |
+|                   7 |              40,000 |               256 |             2.4 |    40,000 |
+|                   5 |              58,000 |               256 |             3.2 |    54,000 |
+|                   7 |              82,000 |               256 |               3 |    52,000 |
+|                  10 |              58,000 |               256 |             3.2 |    53,000 |
+
```
Comment (rockb1017) [inline README.md:302]: perf numbers are handled by other PR. remove from here.

File: README.md [hunk 5]
```diff
@@ -162,11 +193,181 @@ $ helm delete my-splunk-otel-collector
...
+if [ -d "/var/log/pods" ]
+then
+    sudo chgrp -R otel /var/log/pods
+    sudo chmod -R g+rwx /var/log/pods
+    sudo setfacl -Rm d:g:otel:rwx,g:otel:rwx /var/log/pods
+fi
+```
+
+
+## Architecture
```
Comment (rockb1017) [inline README.md:258]: this pertains to collecting logs. so maybe call it "Log Architecture" or something more fitting. 
i am thinking it is better to be added as a separate doc file?

File: README.md [hunk 6]
```diff
@@ -162,11 +193,181 @@ $ helm delete my-splunk-otel-collector
...
+Splunk OpenTelemetry Collector for Kubernetes supports installation using Helm. Read the Prerequisites, Installation and Deployment documentation before you start your deployment of Splunk OpenTelemetry Collector for Kubernetes.
+
+Perform the following steps before you install:
+
+1. Create a minimum of one Splunk platform index. This index events index, which will handle logs.
+If you do not configure this index, Splunk OpenTelemetry Collector for Kubernetes uses the defaults created in your HTTP Event Collector (HEC) token.
+
+2. Create a HEC token if you do not already have one. If you are installing the connector on Splunk Cloud, file a ticket with Splunk Customer Service and they will deploy the indexes for your environment, and generate your HEC token.
+
+### Setup for Non-Root User Group
```
Comment (rockb1017) [inline README.md:212]: remove this part `Setup for Non-Root User Group`

Comment (rockb1017) [reply README.md:212]: it is covered in advanced configuration.

File: README.md [hunk 7]
```diff
@@ -162,11 +193,181 @@ $ helm delete my-splunk-otel-collector
 The command removes all the Kubernetes components associated with the chart and
 deletes the release.
 
-## Advanced Configuration
+### Advanced Configuration
 
 To fully configure the Helm chart, see the [advanced
 configuration](docs/advanced-configuration.md).
 
+## Setup Splunk
```
Comment (rockb1017) [inline README.md:201]: this seems redundant. you already mentioned in pre req with links. I say remove.

File: README.md [hunk 8]
```diff
@@ -162,11 +193,181 @@ $ helm delete my-splunk-otel-collector
...
+## Search for Splunk OpenTelemetry Collector for Kubernetes metadata in Splunk
+Splunk OpenTelemetry Collector for Kubernetes sends events to Splunk which can contain extra meta-data attached to each event. Metadata values such as "k8s.pod.name", "k8s.pod.uid", "k8s.deployment.name","k8s.cluster.name", "k8s.namespace.name", "k8s.node.name", "k8s.pod.start_time", "container_name", "run_id" and "stream" will appear as fields when viewing the event data inside Splunk.
+There are two solutions for running searches in Splunk on meta-data.
+
+* Modify search to use`fieldname::value` instead of `fieldname=value`.
+* Configure `fields.conf` on your downstream Splunk system to have your meta-data fields available to be searched using `fieldname=value`. Example: [fields.conf.example](https://github.com/splunk/sck-otel/blob/main/example/fields.conf.example)
+
+For more information on index time field extraction please view this [guide](https://docs.splunk.com/Documentation/Splunk/latest/Data/Configureindex-timefieldextraction#Where_to_put_the_configuration_changes_in_a_distributed_environment).
+
+## Advanced Configurations for Splunk OpenTelemetry Collector for Kubernetes
```
Comment (rockb1017) [inline README.md:324]: remove everything that is added below.

**Review by rockb1017 — CHANGES_REQUESTED** [review]
I see lot of reference to old repo. and much of the content itself doesn't seem fitting here. left comments. Thanks!

File: README.md (hunk 1 above)
Comment (andrewy-splunk) [reply README.md:111]: Added

File: README.md (hunk 5 above)
Comment (andrewy-splunk) [reply README.md]: updated - i think it fits here rather than a different section.

File: README.md (hunk 6 above)
Comment (andrewy-splunk) [reply README.md]: removed

File: README.md (hunk 8 above)
Comment (andrewy-splunk) [reply README.md]: removed - i think. take a look in case i removed something wrong

**Review by andrewy-splunk — COMMENTED** [review]
made changes based on @rockb1017 's comments

File: README.md (hunk 7 above)
Comment (andrewy-splunk) [reply README.md]: removed

File: README.md (hunk 4 above)
Comment (andrewy-splunk) [reply README.md]: removed

File: README.md (hunk 3 above)
Comment (andrewy-splunk) [reply README.md]: moved and deleted - i think this is what you meant?

File: README.md (hunk 2 above)
Comment (andrewy-splunk) [reply README.md]: moved - it shouldn't affect the link though

File: README.md (hunk 1 above)
Comment (andrewy-splunk) [reply README.md:111]: added

File: README.md [hunk 9]
```diff
@@ -159,13 +174,75 @@ To uninstall/delete a deployment with name `my-splunk-otel-collector`:
 $ helm delete my-splunk-otel-collector
 ```
 
-The command removes all the Kubernetes components associated with the chart and
-deletes the release.
+## Log Architecture
 
-## Advanced Configuration
+Splunk OpenTelemetry Collector for Kubernetes deploys a DaemonSet on each node. And in the DaemonSet, a OpenTelemetry container runs and does the collecting job. Splunk OpenTelemetry Collector for Kubernetes uses the [node logging agent](https://kubernetes.io/docs/concepts/cluster-administration/logging/#using-a-node-logging-agent) method. See the [Kubernetes Logging Architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/) for an overview of the types of Kubernetes logs from which you may wish to collect data as well as information on how to set up those logs.
```
Comment (flands) [inline README.md:179]: Please move this to docs folder and provide link here. The README should have basic getting started information and product docs or docs folder should have additional information.

File: README.md [hunk 10]
```diff
@@ -159,13 +174,75 @@ To uninstall/delete a deployment with name `my-splunk-otel-collector`:
...
+## Maintenance And Support
+Splunk OpenTelemetry Collector for Kubernetes is supported through Splunk Support assuming the customer has a current Splunk support entitlement ([Splunk Support](https://www.splunk.com/en_us/about-splunk/contact-us.html#tabs/tab_parsys_tabs_CustomerSupport_4)). For customers that do not have a current Splunk support entitlement, please search [open and closed issues](https://github.com/splunk/sck-otel/issues?q=is%3Aissue) and create a new issue if not already there.
+The current maintainers of this project are the DataEdge team at Splunk.
+
+## Contributing
+We welcome feedback and contributions from the community! Please see our ([contribution guidelines](https://github.com/splunk/sck-otel/blob/main/CONTRIBUTING.md)) for more information on how to get involved. PR contributions require acceptance of both the code of conduct and the contributor license agreement.
+
+## Upgrading
+## v0.2.x -> v0.3.0
+If using `.Values.configOverride` and have expressions that refer log record, double up `$` characters for those expressions. [Expressions](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/docs/types/expression.md)
```
Comment (dmitryax) [inline README.md:244]: This section is unrelated to this chart. Please remove

File: README.md [hunk 11]
```diff
@@ -103,18 +100,36 @@ require additional configurations applied to
 ### Prerequisites
 
 The following prerequisites are required to use the helm chart:
+- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported) 
+- See https://github.com/kubernetes/helm for more information.
```
Comment (dmitryax) [inline README.md:104]: This was a list of prerequisites: installed Helm 3 and Kubernetes cluster. It doesn't seem to be a list of prerequisites anymore.

Can we move the following line from the bottom here
- Administrator access to your [Kubernetes cluster](https://kubernetes.io/) and familiarity with your Kubernetes configuration. You must know where your log information is being collected in your Kubernetes deployment.

instead of this extra helm link?

File: README.md [hunk 12]
```diff
@@ -103,18 +100,36 @@ require additional configurations applied to
...
 
-- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported)
-- [Kubernetes cluster](https://kubernetes.io/)
-
-- To send data to [Splunk Enterprise/Cloud](https://docs.splunk.com/Documentation/Splunk/8.2.2/Data/UsetheHTTPEventCollector)
-  - HEC Token
-  - HEC Endpoint
+#### To send data to Splunk Enterprise/Cloud:
```
Comment (dmitryax) [inline README.md:106]: If we transform it to header, can we do the same for Splunk Observability Cloud for consistency?

File: README.md [hunk 13]
```diff
@@ -103,18 +100,36 @@ require additional configurations applied to
...
   - [Splunk Realm](https://dev.splunk.com/observability/docs/realms_in_endpoints/)
 
+
+- Administrator access to your [Kubernetes cluster](https://kubernetes.io/) and familiarity with your Kubernetes configuration. You must know where your log information is being collected in your Kubernetes deployment.
+
+### To install and configure defaults with Helm:
+
+```bash
+helm install my-splunk-otel-collector --set="splunkPlatform.endpoint=127.0.0.1:8088,splunkPlatform.token=xxxxxx,splunkPlatform.metrics_index=k8s-metrics,splunkPlatform.index=main,clusterName=my-cluster" splunk-otel-collector-chart/splunk-otel-collector
+```
```
Comment (dmitryax) [inline README.md:124]: We already have _How to install_ section. Can you it it there?

File: README.md [hunk 14]
```diff
@@ -159,13 +174,75 @@ To uninstall/delete a deployment with name `my-splunk-otel-collector`:
...
+* [OpenTelemetry attributes processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/attributesprocessor) to modify attributes of logs
+* Splunk OpenTelemetry Collector for Kubernetes uses multiple operators from [OpenTelemetry log collection operators](https://github.com/open-telemetry/opentelemetry-log-collection/tree/main/docs/operators) like regex_parser, recombine, restructure, json_parser, metadata for enriching logs with metadata and transforming/standardizing logs and metadata from various container runtimes
+* [OpenTelemetry Splunk HEC exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/splunkhecexporter): to send logs to Splunk HTTP Event Collector. The [HTTP Event Collector](http://dev.splunk.com/view/event-collector/SP-CAAAE6M) collects all data sent to Splunk for indexing.
+
+### Add logs from different Kubernetes distributions and container runtimes like(docker, cri-o, containerd)
+
+Select the proper container runtime for your Kubernetes distribution.
+
+[Example](https://github.com/splunk/sck-otel/blob/main/charts/sck-otel/values.yaml#L47)
```
Comment (dmitryax) [inline README.md:206]: Please update all the link to this helm chart

File: README.md [hunk 15]
```diff
@@ -159,13 +174,75 @@ To uninstall/delete a deployment with name `my-splunk-otel-collector`:
...
+
+### Tweak Performance/resources used by Splunk OpenTelemetry Collector for Kubernetes
+If you want to tweak performance/cpu and memory resources used by  Splunk OpenTelemetry Collector for Kubernetes change the available cpu and memory for the Opentelemtry Agent by configuring resources:limits:cpu and resources:limits:memory in the values.yaml file used to deploy Splunk OpenTelemetry Collector for Kubernetes.
+
+[Example](https://github.com/splunk/sck-otel/blob/main/charts/sck-otel/values.yaml#L143)
+
+## Maintenance And Support
+Splunk OpenTelemetry Collector for Kubernetes is supported through Splunk Support assuming the customer has a current Splunk support entitlement ([Splunk Support](https://www.splunk.com/en_us/about-splunk/contact-us.html#tabs/tab_parsys_tabs_CustomerSupport_4)). For customers that do not have a current Splunk support entitlement, please search [open and closed issues](https://github.com/splunk/sck-otel/issues?q=is%3Aissue) and create a new issue if not already there.
+The current maintainers of this project are the DataEdge team at Splunk.
```
Comment (dmitryax) [inline README.md:237]: This is already covered in troubleshooting doc. I think it can be removed, or need to be updated to cover both Observability and Enterprise

File: README.md [hunk 16]
```diff
@@ -159,13 +174,75 @@ To uninstall/delete a deployment with name `my-splunk-otel-collector`:
...
+* [OpenTelemetry resource processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/resourceprocessor) to apply changes on resource attributes
+* [OpenTelemetry attributes processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/attributesprocessor) to modify attributes of logs
+* Splunk OpenTelemetry Collector for Kubernetes uses multiple operators from [OpenTelemetry log collection operators](https://github.com/open-telemetry/opentelemetry-log-collection/tree/main/docs/operators) like regex_parser, recombine, restructure, json_parser, metadata for enriching logs with metadata and transforming/standardizing logs and metadata from various container runtimes
+* [OpenTelemetry Splunk HEC exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/splunkhecexporter): to send logs to Splunk HTTP Event Collector. The [HTTP Event Collector](http://dev.splunk.com/view/event-collector/SP-CAAAE6M) collects all data sent to Splunk for indexing.
+
+### Add logs from different Kubernetes distributions and container runtimes like(docker, cri-o, containerd)
```
Comment (dmitryax) [inline README.md:202]: This section and others below should go to [Advanced configuration doc](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md)
